src/webmon_app/reporting/reporting_app/settings/base.py

```diff
@@ -281,6 +281,12 @@ def validate_ldap_settings(server_uri, user_dn_template):
 DASMON_PLOT_TIME_RANGE = 2 * 60 * 60
 # Max number of old PV points to show when there were no points in defined time range
 DASMON_NUMBER_OF_OLD_PTS = 20
+# Time range in number of hours when showing list of latest runs
```

Comment:
Slightly different way to get to the same place:
```py
LATEST_RUNS_TIME_RANGE_HOURS_DEFAULT = 12.
try:
    LATEST_RUNS_TIME_RANGE_HOURS = float(environ.get("LATEST_RUNS_TIME_RANGE_HOURS", LATEST_RUNS_TIME_RANGE_HOURS_DEFAULT))
except ValueError:
    LATEST_RUNS_TIME_RANGE_HOURS = LATEST_RUNS_TIME_RANGE_HOURS_DEFAULT
del LATEST_RUNS_TIME_RANGE_HOURS_DEFAULT
```
I don't know if it is clearer or not